app/frontend/containers/EmailTargetView/EmailTargetView.js

```diff
@@ -0,0 +1,211 @@
+            name='select-fund' options={this.props.pensionFunds} />
+        </div>
+
+
+        <div className='form__group'>
+          <Input
+            name='email_subject'
+            value={this.props.subject}
+            label={<FormattedMessage id="email_target.form.subject" defaultMessage="Subejct (default)" />}
+            onChange={(value) => this.props.changeSubject(value)} />
+        </div>
+
+        <div className='form__group'>
+          <Input
+            name='name'
+            label={<FormattedMessage id="email_target.form.your_name" defaultMessage="Your name (default)" />}
+            value={this.props.name}
+            required={true}
+            onChange={(value) => this.props.changeName(value)} />
+
+        </div>
+
+        <div className='form__group'>
+          <Input
+            name='email'
+            label={<FormattedMessage id="email_target.form.your_email" defaultMessage="Your email (default)" />}
+            value={this.props.email}
+            required={true}
+            onChange={(value) => this.props.changeEmail(value) } />
+        </div>
+
+        <div className='form__group'>
+          <div className='email__target-body'>
+
+          <textarea
+            name='email_body'
+            value={this.props.body}
+            onChange={(event) => this.props.changeBody(event.currentTarget.value)}
+            maxLength="9999">
+          </textarea>
+          </div>
+        </div>
+
+        <div className='form__group'>
+          <Button disabled={this.props.isSubmitting} className='button action-form__submit-button'>
+            <FormattedMessage id="email_target.form.send_email" defaultMessage="Send email (default)" />
+          </Button>
+        </div>
+      </form>
+      </div>
+
+    </div>
+    );
+  }
+}
+
+type EmailTargetType = {
+  emailBody: string,
+  emailSubject: string,
+  country: string,
+  email: string,
+  name: string,
+  pensionFunds: Array<string>,
+  isSubmitting: boolean,
+  to: string,
+  fundId: string,
+  fund: string,
+  fundContact: string,
+  fundEmail: string,
+  page: string,
+}
+
+type OwnState = {
+  emailTarget: EmailTargetType,
+}
+
+export const mapStateToProps = (state: OwnState) => ({
+  body: state.emailTarget.emailBody,
+  subject: state.emailTarget.emailSubject,
+  country: state.emailTarget.country,
+  email: state.emailTarget.email,
+  name: state.emailTarget.name,
+  pensionFunds: state.emailTarget.pensionFunds,
+  fundId: state.emailTarget.fundId,
+  fund: state.emailTarget.fund,
+  fundContact: state.emailTarget.fundContact,
+  fundEmail: state.emailTarget.fundEmail,
+  to: state.emailTarget.to,
+  page: state.emailTarget.page,
+  isSubmitting: state.emailTarget.isSubmitting,
+});
+
+export const mapDispatchToProps = (dispatch: Dispatch<*>) => ({
+  changeBody: (body: string) => dispatch(changeBody(body)),
+
+  changeCountry: (country: string) => {
+    dispatch(changeCountry(country));
+  },
```

Comment:
Not really important but this returns undefined (dispatch returns the action dispatched). If you ever wanted to test `mapDispatchToProps` you might want to be able to get the return value here.